libs/model/src/main/kotlin/de/cyface/model/Track.kt

```diff
@@ -31,29 +30,29 @@ import java.util.stream.Collectors
  * when the very last locations is reached.
  *
  * @author Armin Schnabel
- * @version 2.0.1
+ * @version 2.1.0
  * @since 1.0.0
  */
 class Track : Serializable {
     /**
      * The list of `GeoLocationRecord`s collected for this `Track` ordered by timestamp.
      */
-    private lateinit var locationRecords: List<RawRecord>
+    private lateinit var locationRecords: MutableList<RawRecord>
 
     /**
      * The list of accelerations for this `Track` ordered by timestamp. Unit: m/s².
      */
-    private lateinit var accelerations: List<Point3DImpl>
+    private lateinit var accelerations: MutableList<Point3DImpl>
 
     /**
      * The list of rotations for this `Track` ordered by timestamp. Unit: rad/s.
      */
-    private lateinit var rotations: List<Point3DImpl>
+    private lateinit var rotations: MutableList<Point3DImpl>
 
     /**
      * The list of directions for this `Track` ordered by timestamp. Unit. micro-Tesla (uT).
      */
-    private lateinit var directions: List<Point3DImpl>
+    private lateinit var directions: MutableList<Point3DImpl>
```

Comment:
before this was converted to Kotlin:

```
    private List<RawRecord> locationRecords;
    private List<Point3DImpl> accelerations;
    private List<Point3DImpl> rotations;
    private List<Point3DImpl> directions;

    public List<RawRecord> getLocationRecords() {
        return Collections.unmodifiableList(locationRecords);
    }
    public List<Point3DImpl> getAccelerations() {
        return Collections.unmodifiableList(accelerations);
    }
    public List<Point3DImpl> getRotations() {
        return Collections.unmodifiableList(rotations);
    }
    public List<Point3DImpl> getDirections() {
        return Collections.unmodifiableList(directions);
    }
```

And the backend is accessing the mutable lists like `track.accelerations` everywhere.

I could either
- make `accelerations` etc a List instead of MutableList and add a getMutableAccelerations() method for the instances where the backend actually needs to modify the points inside a track
- or like here: don't refactor the backend code and make the properties mutable and remove the getAccelerations() etc methods from the java code.